docs/pages/web/sheetmobile.js

```diff
@@ -0,0 +1,326 @@
+// @flow strict
+import { type Node } from 'react';
+import { SlimBanner } from 'gestalt';
+import AccessibilitySection from '../../docs-components/AccessibilitySection.js';
+import { multipledocgen, type DocGen } from '../../docs-components/docgen.js';
+import GeneratedPropTable from '../../docs-components/GeneratedPropTable.js';
+import MainSection from '../../docs-components/MainSection.js';
+import Page from '../../docs-components/Page.js';
+import PageHeader from '../../docs-components/PageHeader.js';
+import QualityChecklist from '../../docs-components/QualityChecklist.js';
+import SandpackExample from '../../docs-components/SandpackExample.js';
+import animation from '../../examples/sheetmobile/animation.js';
+import autoSize from '../../examples/sheetmobile/autoSize.js';
+import defaultSize from '../../examples/sheetmobile/defaultSize.js';
+import dismissButtonHeader from '../../examples/sheetmobile/dismissButtonHeader.js';
+import footer from '../../examples/sheetmobile/footer.js';
+import fullSize from '../../examples/sheetmobile/fullSize.js';
+import main from '../../examples/sheetmobile/main.js';
+import navigationHeader from '../../examples/sheetmobile/navigationHeader.js';
+import outsideClick from '../../examples/sheetmobile/outsideClick.js';
+import textOnlyHeader from '../../examples/sheetmobile/textOnlyHeader.js';
+import withPrimaryActionHeader from '../../examples/sheetmobile/withPrimaryActionHeader.js';
+
+export default function SheetMobilePage({
+  generatedDocGen,
+}: {|
+  generatedDocGen: {| [string]: DocGen |},
+|}): Node {
+  return (
+    <Page title={generatedDocGen?.SheetMobile.displayName}>
+      <PageHeader
+        badge="pilot"
+        name={generatedDocGen?.SheetMobile.displayName}
+        description={generatedDocGen?.SheetMobile.description}
+        slimBanner={
+          <SlimBanner
+            type="warning"
+            iconAccessibilityLabel="Warning message"
+            message="SheetMobile is in pilot phase. Expect development and design iteration and breaking API changes as well as further documentation development."
+          />
+        }
+      >
+        <SandpackExample code={main} hideEditor name="Main SheetMobile example" />
+      </PageHeader>
+
+      <GeneratedPropTable generatedDocGen={generatedDocGen?.SheetMobile} />
+
+      <MainSection name="Usage guidelines">
+        <MainSection.Subsection columns={2}>
+          <MainSection.Card
+            cardSize="md"
+            type="do"
+            title="When to use"
+            description={`
+- Use a Partial Sheet when the content of the Sheet compliments the content of the primary screen behind.
+- Use a Full Sheet when the content of the Sheet does not need to reference the primary screen behind. Usually serving as a lightweight way to complete actions or move through a flow.
+- Use in conjunction with other Gestalt components as a means to make them mobile-friendly e.g. Dropdown or Popover.`}
```

Comment:
Is this something we want to explicitly encourage? Are we going to provide this UI under-the-hood for those components on mobile?

Reply:
This is something to clarify in guidelines as if it's just for designers it might be good to specify,,,